gamestonk_terminal/fred/README.md

```diff
@@ -0,0 +1,46 @@
+# fred
+
+The purpose of fred is to have the ability to look at economic data.
+
+* [GDP](#GDP)
+    * Get GDP data
+* [cust](#cust)
+    * Get custom fred data  
+
+## GDP <a name="GDP"></a>
+```text
+usage: GDP [-n N_TO_GET] [-s START_DATE] [-p PLOT]]
+```
+Gets GDP data.  Data is released quarterly
+* -n : Number of points to display.  If unspecified, it will display all available.
+* -s : Start Date.  Format m/d/y. First point to get
+* -p : Option to plot.  Defaults to False
+
+To get the last 20 reported GDP values:
+````
+GDP -n 20
+````
+To plot all GDP data since 1/1/2010:
+````
+GDP -s 1/1/2010 -p True
+````
+
+
+## cust <a name="cust"></a>
+
+The fred database has many different datasets freely available.  They each have their own custom SERIES_ID which can be obtained through the website.
+
+````
+usage: cust [-id] [-s START_DATE] [-p PLOT] [-disp DISP]
+````
+* -id : Series ID for FRED data.  Required argument
+* -s : Start data to acquire data.  Defaults to 1/1/2020
+* -p : Option to plot. Defaults to False
+* -disp : Option to print data to console.  Defaults to True.
```

Comment:
If the default of print data to console is set to True, shouldn't the flag be called "--nodisp"? 

Again, I would probably add plot and stdout print as default, but I'm not knowledgeable enough here to have a saying, so I'll take your input.

Reply:
Ill set the default print to False (along with plotting true). 

This way the user can see the numbers if they want, but won't get their console just spewing numbers.

Reply:
Yea, I guess it depends on the number of data.

E.g. viewing historical data, doesn't make sense to output this by default. Once using a prediction algorithm to predict next 5 market days, it makes sense to print those.